modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -493,3 +493,161 @@ func (suite *KeeperTestSuite) TestQueryCounterpartyAddress() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestQueryFeeEnabledChannels() {
+	var (
+		req                   *types.QueryFeeEnabledChannelsRequest
+		expFeeEnabledChannels []types.FeeEnabledChannel
+	)
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success",
+			func() {},
+			true,
+		},
+		{
+			"success, empty pagination",
+			func() {
+				req = &types.QueryFeeEnabledChannelsRequest{}
+			},
+			true,
+		},
+		{
+			"success, with multiple fee enabled channels",
```

Comment:
Is the test description correct? I see below that only one channel is appended to `expFeeEnabledChannels`...

Reply:
Below in the body of the test func, the first (default) expected channel is added to `expFeeEnabledChannels` on line 579, then `malleate()` is called after, and here we're just appending an additional expected channel.
Because `expFeeEnabledChannels` is defined at the top level scope of the test function we can mutate it in `malleate`

Reply:
Ah, I see now, thanks. Should we make it explicit in the description that this test does not use pagination? Something like `success, no pagination with multiple fee enabled channels`?

Reply:
Sure, can do, I'm going to close this PR and re-open with main as the target. I don't want to spoil the commit history if I change the base branch on this PR.

Reply:
Just taking a second look at this now since I created a new branch for main. This test does use pagination. The default request that's built inside the body of the test function is used:

See line 581

```
req = &types.QueryFeeEnabledChannelsRequest{
	Pagination: &query.PageRequest{
		Limit:      5,
		CountTotal: false,
	},
	QueryHeight: 0,
}
```